### ISSUE TITLE
Cannot make dev-env

### ISSUE THREAD

**Issue by vvarg229** [nspcc-dev/neofs-node#2361]
For tests we use https://github.com/nspcc-dev/neofs-dev-env
After this PR https://github.com/nspcc-dev/neofs-node/pull/2344 (commit a65293db977e0dcad152960da4845aacf19b9ffe) when creating the dev-env neofs-ir container it fails the healthcheck:
For exapmle:
https://github.com/vvarg229/neofs-node/actions/runs/5106970154/jobs/9179502579
`ERROR: for sn-healthcheck  Container "b4c9be1e8134" is unhealthy.
`

What healthcheck does:

```
#!/bin/sh

/neofs-cli control healthcheck \
     --endpoint "$NEOFS_IR_CONTROL_GRPC_ENDPOINT" \
     --wallet /wallet01.key --ir


```

The result of manually verbose running the healthcheck (for https://github.com/vvarg229/neofs-node/actions/runs/5122299086/jobs/9215220099 run):

```
varg@burzum:~/work/actions-runner/_work/neofs-node/neofs-node/neofs-dev-env$ ./vendor/neofs-cli --verbose control healthcheck --endpoint 127.0.0.1:16512 --wallet ./services/ir/wallet01.json --ir --timeout 30s
Using default wallet address
Enter password > 
Set request timeout to 30s.
can't create API client: can't init SDK client: open gRPC connection: gRPC dial: context deadline exceeded

varg@burzum:~/work/actions-runner/_work/neofs-node/neofs-node/neofs-dev-env$ docker ps
CONTAINER ID   IMAGE                             COMMAND                  CREATED              STATUS                          PORTS                          NAMES
3f6d87d8c5e0   nspccdev/neofs-storage:a65293db   "neofs-node"             About a minute ago   Restarting (2) 44 seconds ago                                  s02
0f3dc5b97cd4   nspccdev/neofs-storage:a65293db   "neofs-node"             About a minute ago   Restarting (2) 42 seconds ago                                  s04
8fcec6a1dbca   nspccdev/neofs-storage:a65293db   "neofs-node"             About a minute ago   Restarting (2) 42 seconds ago                                  s03
39dada60ceb9   nspccdev/neofs-storage:a65293db   "neofs-node"             About a minute ago   Restarting (2) 43 seconds ago                                  s01
c3b906545c65   nspccdev/neofs-ir:a65293db        "neofs-ir"               About a minute ago   Up About a minute (healthy)                                    ir01
d6ecefbc5d44   nats:2.7.2                        "/nats-server -c /et…"   About a minute ago   Up About a minute               4222/tcp, 6222/tcp, 8222/tcp   nats
e763ac19fcf0   nspccdev/neo-go:0.101.1           "/usr/bin/privnet-en…"   2 minutes ago        Up 2 minutes                                                   morph_chain
5290d94d743a   nspccdev/neo-go:0.101.1           "/usr/bin/privnet-en…"   2 minutes ago        Up 2 minutes                                                   main_chain
a62dee27ab80   debian:10                         "/bin/sleep infinity"    2 minutes ago        Up 2 minutes                                                   bastion

```

Logs:
[c3b906545c65.log](https://github.com/nspcc-dev/neofs-node/files/11602364/c3b906545c65.log)

If I roll back to the previous commit, the dev-env is makes successfully:
https://github.com/vvarg229/neofs-node/actions/runs/5111910127/jobs/9189388442

## Expected Behavior
https://github.com/vvarg229/neofs-node/actions/runs/5111910127/jobs/9189388442

## Current Behavior
https://github.com/vvarg229/neofs-node/actions/runs/5106970154/jobs/9179502579

## Possible Solution
I think the problem is somewhere in the configuration, probably in the config file:
https://github.com/nspcc-dev/neofs-dev-env/blob/master/services/ir/.ir.env

I tried to localize it even more accurately, but unfortunately I could not.

## Steps to Reproduce (for bugs)
Run the system tests.
Or run a specially created debug job:
https://github.com/vvarg229/neofs-node/pull/13
Or `make prepare-test-env`

## Context
This issue blocks the running of system tests and is most likely a configuration related bug.

## Regression
Yes
https://github.com/nspcc-dev/neofs-node/pull/2344

## Your Environment
ubuntu-latest GitHub runners
and my laptop as GitHub runner:
`Linux burzum 6.2.0-20-generic #20-Ubuntu SMP PREEMPT_DYNAMIC Thu Apr  6 07:48:48 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
`

**Comment by vvarg229** [comment]
@notimetoname PR https://github.com/nspcc-dev/neofs-node/pull/2344 you did.
Mb you can help?

**Comment by vvarg229** [comment]
> @vvarg229, after #2344 there was a little (but almost immediate) fix for it: #2358. Have you tried it? The current master is working on my local dev-env.
@notimetoname Yes, I tried it
No, it doesn't  works:
https://github.com/vvarg229/neofs-node/actions/runs/5141841364/jobs/9254811870

Please send the .env file you are using. Could the trouble be in one of the other components?

**Comment by roman-khimov** [comment]
>  non-fixed neofs-cli version

It's a feature, we're testing the product of revision X, `neofs-cli` is a part of this product. If we're to require some specific compatibility tests that would be specific compatibility tests with these tests getting some static old version they need.
